homeassistant/components/zwave/__init__.py

```diff
@@ -600,14 +601,36 @@ def start_zwave(_service_or_event):
     return True
 
 
-class ZWaveDeviceEntity:
+class ZWaveDeviceEntity(Entity):
```

Comment:
As I understand Python's MRO, this should cause no changes, as long as Entity was being mixed in eventually. In the example given, both before and after, methods would be checked from `ZwaveDimmer` -> `zwave.ZWaveDeviceEntity` -> `Light` -> `Entity`. Entity is now pulled in twice, which makes this a diamond inheritance pattern. There's a post about it [here](http://stackoverflow.com/a/1848647) that explains it. Since we're on Python 3, all objects are the new-style.